index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <link rel="stylesheet" type="text/css" href="style.css" />
+    </head>
+
+    <body class="container">
+        <header>
+            <div class="header">
+                <div class="logo">
+                   <a href=""><img src="My Logo.png" alt="" class="logo-img" /></a> 
+                </div>
+                <nav>
+                    <div class="menu">
+                        <img src="Union.svg" alt="">
+                    </div>
+                </nav>
+            </div>
+        </header>
```

Comment:
- [ ] ***[OPTIONAL]*** As suggested by the previous reviewer, kindly adjust the section background images so that it covers the entire section to match the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).
- ***Figma Design***
![1](https://user-images.githubusercontent.com/60083437/235319904-d2de908e-de99-4cdd-b70c-c3fec7eb77c7.png)

- ***Your Design***
![2](https://user-images.githubusercontent.com/60083437/235319912-698ce102-f050-4ff1-bfe5-b2a04257b8e8.png)